### ISSUE TITLE
Crash sensitivty

### ISSUE THREAD

**Issue by shazz** [salem-ok/PySprint#61]
In the original game, it is easier to crash against the borders (mostly every time at full speed) than in pysprint

**Comment by salem-ok** [comment]
Thanks @shazz 

- I'll create a separate issue for the rebound and look into
- For the crash, there are many parameters you can use to adjust that in pysprint_car.py. I've been tweaking it a lot. My perception with the original is that you crash at max speed, only after being at high speed for a few seconds, so I've implemented that parameter, but have not been able to perfectly calibrate it)

generally if you lower a bit the crash_certainty_threshold you will crash a lot more.

````python
    #Collision Settings
    diagonal_detection_tolerance = 2
    vector_simulation_length = 10
    side_detection_tolerance = 7
    max_speed_crash_threshold = 4000#3000
    collision_area_threshold = 80#75
    #Threshold over which theer is a higher chance to crash
    speed_crash_probability_threshold = 0.85
    #% increase of probability to crash if condition is true
    speed_crash_probability_penalty = 1.2
    sensitive_border_crash_probability_penalty = 1.4
    #Max Random number drawn to calculate Crash probability
    crash_random_max = 60
    crash_certainty_treshold = 85#80
````

**Comment by salem-ok** [comment]
@shazz I've adjusted the parameters to make the crash more prone to happen. Let me knwo if it feels more realistic to you now!
